src/config.ts

```diff
@@ -22,9 +23,20 @@ export const installationConfig = rt.Record({
 export type InstallationConfig = rt.Static<typeof installationConfig>;
 
 export function makeInstallationConfig(): rt.Result<InstallationConfig> {
+  let pulumiVersion = getInput('pulumi-version') || '^3';
```

Comment:
Should we error out early if both `pulumi-version` and `pulumi-version-file` are given?

Reply:
Good call! I'll update it.